rust/src/tx_builder.rs

```diff
@@ -593,7 +593,10 @@ impl TransactionBuilder {
                         if potential_pure_above_minimum {
                             new_fee = new_fee.checked_add(&additional_fee)?;
                             change_left = Value::zero();
-                            self.add_output(&TransactionOutput::new(address, &potential_pure_value));
+                            match self.add_output(&TransactionOutput::new(address, &potential_pure_value)) {
+                              Err(e) => panic!(e),
+                              Ok(_) => {}
+                            };
```

Comment:
Because the function `add_change_if_needed` returns a `Result<..., JsError>` - the recommended way to deal with functions like this, that complain about unused return result is to use the `?` unwrapper that will fail-fast to propagate the err in case it's returned by the called function.

e.g.
```
self.add_output(&TransactionOutput::new(address, &potential_pure_value))?;
```

This particular line is also being fixed in a parallel PR:

https://github.com/Emurgo/cardano-serialization-lib/pull/231/files#diff-67896f615f1e60f7474eb6194c0614a005522dc35fb07046855d77260954b972R775